specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerAppsLabelHistory.json

```diff
@@ -0,0 +1,215 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-10-02-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/labelHistory": {
+      "get": {
+        "tags": [
+          "ContainerAppsLabelHistory"
+        ],
+        "summary": "Get the Label History for a given Container App.",
+        "operationId": "ContainerAppsLabelHistory_ListLabelHistory",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v3 [](http://example.com/codeflow?start=69&length=2)

use the v5 also in this file

Reply:
why in this file you are not using v5?

Reply:
For many existing rosource types, we are still using v3. But if v5 is required, I can update to v5. Please help approve  and merge https://github.com/Azure/azure-rest-api-specs/pull/31545. Then it will auto commit to this PR.

Reply:
thanks, this helps having an uniform version in all files.

Reply:
updated